server/cdc/listener.go

```diff
@@ -46,7 +46,7 @@ func (p *Publisher) OnCommit(ctx context.Context, tx transaction.Tx, listener kv
 		return err
 	}
 
-	td := internal.NewTableDataWithEncoding(json, int32(internal.JsonEncoding))
+	td := internal.NewTableDataWithVersion(json, 1)
```

Comment:
Lets make the magic number a const from the `internal` package. `internal.DefaultVersion`